rest-api-spec/src/main/resources/rest-api-spec/api/ml.infer_trained_model_deployment.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ml.infer_trained_model_deployment":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-infer-trained-model-deployment.html",
+      "description":"Evaluate a trained model."
```

Comment:
Confirming this is the proper description for this API

Reply:
Yes 'evaluating' a model can be considered another way of saying 'performing inference' or even 'calling forward (propogation)'

Thanks for the review